portal/urls.py

```diff
@@ -0,0 +1,28 @@
+from django.urls import include, path
+from rest_framework import routers
+
+from portal.views import (
+    CreatePoll,
+    CreatePollOptions,
+    CreatePollVote,
+    RetrievePolls,
+    RetrievePollVotes,
+    UpdatePoll,
+    UpdatePollOption,
+    UpdatePollVote,
+)
+
+
+router = routers.DefaultRouter()
+
+router.register(r"polls/view", RetrievePolls)
+urlpatterns = [
+    path("", include(router.urls)),
```

Comment:
It doesn't really matter, but I tend to prefer splitting the router and regular URLs like we do in [OHQ](https://github.com/pennlabs/office-hours-queue/blob/master/backend/ohq/urls.py)